setup.py

```diff
@@ -67,6 +67,7 @@ def run_tests(self):
     entry_points={
         'console_scripts': ['labctl=lab.app.__main__:main'],
         'labctl': [
+            'envs=lab.app.environments:EnvsLister',
```

Comment:
@dtkerr can we swap to the `env` instead of `show` and add a warning for the latter usage (so don't remove it just prepare for deprecation)?

@vodik I like `envs` myself but I won't get too picky.
@dtkerr If you add that I think this can be merged :)